config/core/resources/eventtype.yaml

```diff
@@ -155,9 +155,14 @@ spec:
       type: string
       jsonPath: ".status.conditions[?(@.type==\"Ready\")].reason"
   - <<: *version
-    name: v1beta2
+    name: v1beta1
     served: true
     storage: false
+    # This indicates the v1beta1 version of the custom resource is deprecated.
+    # API requests to this version receive a warning header in the server response.
+    deprecated: true
+    # This overrides the default warning returned to API clients making v1alpha1 API requests.
+    deprecationWarning: "eventing.knative.dev/v1beta1 EventType is deprecated; see http://knative.dev for instructions to migrate to eventing.knative.dev/v1beta2 EventType"
```

Comment:
Do we have a more specific link to migrate EventTypes specifically?

Reply:
updated to a bit more specific page on the docs